spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/NamedSecretNormalizedSource.java

```diff
@@ -25,14 +25,14 @@
  */
 public final class NamedSecretNormalizedSource extends NormalizedSource {
 
-	private final String prefix;
+	private final ConfigUtils.Prefix prefix;
 
-	public NamedSecretNormalizedSource(String name, String namespace, boolean failFast, String prefix) {
+	public NamedSecretNormalizedSource(String name, String namespace, boolean failFast, ConfigUtils.Prefix prefix) {
```

Comment:
Would it be worth having a constructor that doesn't take a `Prefix`?

There are several places where we create a new instance but are just passing `UNSET` as the prefix, seems easier to create an instance without specifying a prefix we just automatically set it to `UNSET`.  

Also using/specifying a prefix is optional so it makes sense to have a constructor that doesn't take one.

(This also applies to the Config Map version as well)

Reply:
done